sphinx/views.py

```diff
@@ -1,14 +1,32 @@
+import os
 import json
+import base64
+import re
 
 from django.shortcuts import render
+from django.utils.safestring import mark_safe
 
+from core.models import Board, School, Standard, Subject, Question, Chapter, QuestionTag, \
+    QuestionSubpart
 from core.data_models.question import build_question_subpart_from_data
+from core.utils.references import EdgeSpecialTags
 from core.utils.json import OpenShikshaJsonResponse
 from croupier.constraints import SubpartVariableConstraints
 from croupier.croupier_api import deal_subpart
 from sphinx.urlnames import SphinxUrlNames
-from sphinx.view_models import Tags
 
+HOME_DIR = os.path.expanduser('~')
```

Comment:
saving the files via the filesystem assumes that cabinet is running on the same machine, but this is not the case. In the deployment, cabinet is running inside another docker container, so you can't even access its filesystem. You should use a PUT request to cabinet to save a file. We use the same pattern to save submission files.